internal/db/schema/migrations/postgres/14/01_event.up.sql

```diff
@@ -0,0 +1,206 @@
+│ public_id       │          ╱├──────────────────┤   
+│ scope_id        │┼────────○─│config_id         │   
+│                 │          ╲│event_type        │   
+│                 │           │                  │   
+└─────────────────┘           └──────────────────┘   
+         ┼                              ┼            
+         ┼                              │            
+         │                              │            
+         │                             ╱│╲           
+         ○                     ┌─────────────────┐   
+        ╱│╲                   ╱│ event_type_enm  │   
+┌─────────────────┐     ┌──────├─────────────────┤   
+│   event_sink    │     │     ╲│                 │   
+├─────────────────┤     │      └─────────────────┘   
+│public_id        │┼────┤                            
+│config_id        │     │     ┌─────────────────────┐
+│                 │     │    ╱│ event_sink_type_enm │
+└─────────────────┘     ├─────├─────────────────────┤
+         ┼              │    ╲│                     │
+         │              │     └─────────────────────┘
+         │              │     ┌─────────────────┐    
+         ○              │    ╱│event_format_type│    
+        ╱│╲             └─────├─────────────────┤    
+┌─────────────────┐          ╲│                 │    
+│ event_file_sink │           └─────────────────┘    
+├─────────────────┤                                  
+│ public_id       │                                  
+│ sink_id         │                                  
+│ path            │                                  
+│ file_name       │                                  
+│ rotate_bytes    │                                  
+│ rotate_duration │                                  
+│ rotate_max_files│                                  
+└─────────────────┘                                                                                      
+*/
+
+create table event_type_enm (
+    name text primary key
+        constraint only_predefined_event_types_allowed
+        check (
+            name in (
+                'every',
+                'error',
+                'audit',
+                'observation',
+                'system'
+            )
+        )
+);
+comment on table event_type_enm is
+'event_type_enm is an enumeration table for the valid event types within the '
+'the domain';
+
+create table event_sink_type_enm (
+    name text primary key
+        constraint only_predefined_event_sink_types_allowed
+        check (
+            name in (
+                'stderr',
+                'file'
+            )
+        )
+);
+comment on table event_type_enm is
+'event_sink_type_enm is an enumeration table for the valid sink types within the '
+'the domain';
+
+create table event_format_type_enm (
+    name text primary key
+        constraint only_predefined_event_format_types_allowed
+        check (
+            name in (
+                'json',
```

Comment:
given the new hclog formats, we may need to change these to be specific for cloudevents and add some hclog types?